pythonFeed.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+vers=()
+
+echo "Initializing submodule..."
+git submodule update --init --recursive
+cd shared || exit && \
+git checkout feat/version-checking
+if [ -f automated-updates.sh ]; then
+  source automated-updates.sh
+else
+  echo "Check if submodule was loaded; automated-updates.sh is missing"
+  exit 1
+fi
+cd ..
+
+LATEST_VERSION=0
+VERSION_LIST=$(pyenv install --list | sed 's/^  //' | grep '^\d' | grep --invert-match 'dev\|a\|b')
```

Comment:
cimg/base doesn't have pyenv, so we'll need to include that somewhere or use the cimg/python image and update the pyenv version to latest